kurl_proxy/okteto.Dockerfile

```diff
@@ -1,15 +1,13 @@
+# syntax=docker/dockerfile:1.3
 FROM golang:1.17
 
 ENV PROJECTPATH=/go/src/github.com/replicatedhq/kots/kurl_proxy
 WORKDIR $PROJECTPATH
-ADD Makefile ./
-ADD Makefile.build ./
-ADD go.mod ./
-ADD go.sum ./
-ADD cmd ./cmd
+COPY go.mod go.sum ./
+RUN go mod download
 
-RUN make build
+COPY . .
```

Comment:
having this before make build will be a hit to build times when ANY file changes rather than just those that are needed for build.